static/js/attachments_ui.js

```diff
@@ -6,36 +6,37 @@ function delete_attachments(attachment) {
     channel.del({url: '/json/attachments/' + attachment, idempotent: true});
 }
 
+function bytes_to_size(bytes) {
+    var sizes = ['Bytes', 'KB', 'MB', 'GB', 'TB'];
+    if (bytes == 0) return '0 Byte';
```

Comment:
I think we can just use `B` here.  It's written "0 Bytes", not "0 Byte", anyway.